subprojects/packagefiles/lame/meson_options.txt

```diff
@@ -1,4 +1,6 @@
-option('decoder', type : 'boolean', value : false,
-       description : 'Build the decoder')
-option('tools', type : 'boolean', value : false,
+option('decoder', type : 'boolean', value : true,
+       description : 'Build the mpg123 decoder')
+option('iconv', type : 'feature',
+       description : 'Build tools with iconv support')
+option('tools', type : 'feature',
```

Comment:
Is the default value `auto` here? If so, you could make `get_option('decoder').disable_auto_if(not curses_dep.found())` and then use the returned value with `decoder.allowed()` for clarity.

Reply:
feature options default to auto yes. Not sure what decoder has to do with tools. It probably has to remain enabled as there is a dependency listed in the wrap file.